.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+  - repo: local
+    hooks:
+      - id: pre-commit-hook
+        name: 🔄 - Running pre-commit hook...
+        entry: ./pre-commit.sh
+        language: script
+        always_run: true
```

Comment:
Se isso for false, o que acontece mesmo?

Reply:
Não roda, o pre-commit só fica dando skip